staging/src/k8s.io/kube-gen/cmd/client-gen/README.md

```diff
@@ -1,4 +1,4 @@
-See [generating-clientset.md](https://kubernetes.io/docs/devel/generating-clientset.md)
+See [generating-clientset.md](https://github.com/kubernetes/community/blob/master/contributors/devel/generating-clientset.md)
```

Comment:
It should be stable if we ever move or have to redirect things. It's something @thockin created (https://github.com/kubernetes/k8s.io/pull/59) that handles the formatting redirection. Right now it's a noop, but it's preferable if we're updating it anyways.